kickstarts/partials/post/ruby_install.ks.erb

```diff
@@ -1,23 +1,9 @@
-curl -L https://github.com/postmodern/ruby-install/archive/v0.6.0.tar.gz | tar xvfzC - /tmp
-
-pushd /tmp/ruby-install*/
-make install
-popd
-
-# The ruby-install will result in about 350MiB of files in /usr/local/src.
-# These do not need to be in the image, so put them on a tmpfs.
-mkdir -p /usr/local/src
-mount -t tmpfs tmpfs /usr/local/src
-
 # Note, we hardcode the ruby patch level for now.
 # Once the --update option is available, we could ask
 # ruby-install to "learn" what new rubies are available
 # See: https://github.com/postmodern/ruby-install/issues/175
```

Comment:
Does your change just install the "ruby-install" installer from RPM, or Ruby itself as well? Looking at the Copr I think it's the former, and the installer still downloads and installs ruby itself.

These lines are here so that the Ruby .tar.gz and the build files are not in the image. I was able to reduce the image by 350MB. Note that just deleting these files is not enough as the underlying storage will continue to be allocated.